internal/build/phases_test.go

```diff
@@ -0,0 +1,516 @@
+package build_test
```

Comment:
I wonder if it would be worthwhile to separate out each phase into it's own file and keep a separate corresponding test file? The only real argument for it is that this file has now become quite large.

Reply:
Fair enough!

Reply:
On trying this out, I think it might actually be better to leave everything in the same file. GoLand needs the top level `TestPhases(t *testing.T)` to run the tests with Ctrl+Shift+R. If we give each phase its own file and top level `TestXXXPhase(t *testing.T)`, we have to do the same clunky setup for each phase (as described here: https://github.com/buildpacks/pack/issues/541). I'd rather leave the setup as-is and break up `phases_test.go` when we tackle https://github.com/buildpacks/pack/issues/541.